gnome-extra/cinnamon/cinnamon-4.4.8.ebuild

```diff
@@ -0,0 +1,193 @@
+		dev-python/pyinotify[${PYTHON_USEDEP}]
+		dev-python/pypam[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/setproctitle[${PYTHON_USEDEP}]
+		dev-python/tinycss[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
+		dev-python/xapp[${PYTHON_USEDEP}]
+	')
+
+	x11-themes/gnome-themes-standard
+	x11-themes/adwaita-icon-theme
+
+	>=gnome-extra/nemo-4.0
+	>=gnome-extra/cinnamon-control-center-4.0[networkmanager=]
+	>=gnome-extra/cinnamon-screensaver-4.0
+
+	gnome-extra/polkit-gnome
+
+	nls? ( >=gnome-extra/cinnamon-translations-4.0 )
+"
+DEPEND="${COMMON_DEPEND}
+	dev-util/gtk-doc
+	>=dev-util/intltool-0.4
+	>=sys-devel/gettext-0.17
+	virtual/pkgconfig
+	!!=dev-lang/spidermonkey-1.8.2*
+"
+# libmozjs.so is picked up from /usr/lib while compiling, so block at build-time
+# https://bugs.gentoo.org/show_bug.cgi?id=360413
+
+S="${WORKDIR}/cinnamon-${PV}"
+
+src_prepare() {
+	# Fix backgrounds path as cinnamon doesn't provide them
+	# https://github.com/linuxmint/Cinnamon/issues/3575
+	eapply "${FILESDIR}"/${PN}-3.8.0-gnome-background-compatibility.patch
+
+	# Use wheel group instead of sudo (from Fedora/Arch)
+	# https://github.com/linuxmint/Cinnamon/issues/3576
+	eapply "${FILESDIR}"/${PN}-3.6.6-wheel-sudo.patch
+
+	# Add polkit agent to required components (from Fedora/Arch), bug #523958
+	# https://github.com/linuxmint/Cinnamon/issues/3579
+	sed -i 's/RequiredComponents=\(.*\)$/RequiredComponents=\1polkit-gnome-authentication-agent-1;/' \
+		files/cinnamon*.session.in || die
+
+	# shebang fixing craziness
+	local p
+	for p in $(grep -rl '#!.*python3'); do
+		python_fix_shebang "${p}"
+	done
+
+	eautoreconf
+	gnome2_src_prepare
+}
+
+src_configure() {
+	gnome2_src_configure \
+		--libdir="${EPREFIX}/usr/$(get_libdir)" \
+		--with-ca-certificates="${EPREFIX}/etc/ssl/certs/ca-certificates.crt" \
+		$(use_enable networkmanager) \
+		BROWSER_PLUGIN_DIR="${EPREFIX}/usr/$(get_libdir)/nsbrowser/plugins"
+}
+
+src_install() {
+	gnome2_src_install
+	python_optimize "${ED}"usr/share/cinnamon/
+
+	# Required for gnome-shell on hardened/PaX, bug #398941
+	pax-mark mr "${ED}usr/bin/cinnamon"
+
+	# Doesn't exist on Gentoo, causing this to be a dead symlink
+	rm -f "${ED}etc/xdg/menus/cinnamon-applications-merged" || die
+
+	# Ensure authentication-agent is started, bug #523958
+	# https://github.com/linuxmint/Cinnamon/issues/3579
+	insinto /etc/xdg/autostart/
+	doins "${FILESDIR}"/polkit-cinnamon-authentication-agent-1.desktop
+}
+
+pkg_postinst() {
+	gnome2_pkg_postinst
+
+	if ! has_version 'media-libs/gst-plugins-good:1.0' || \
+	   ! has_version 'media-plugins/gst-plugins-vpx:1.0'; then
+		ewarn "To make use of Cinnamon's built-in screen recording utility,"
+		ewarn "you need to either install media-libs/gst-plugins-good:1.0"
+		ewarn "and media-plugins/gst-plugins-vpx:1.0, or use dconf-editor to change"
+		ewarn "org.cinnamon.recorder/pipeline to what you want to use."
+	fi
+
+	if ! has_version ">=x11-base/xorg-server-1.11"; then
```

Comment:
This is a really old version. Same for others. Still needed?

Reply:
I removed the `x11-base/xorg-server` and `x11-drivers/ati-drivers` since they were positively ancient.

In general, is there a policy about maintaining package specific warnings/blockers? Is it ok to remove them once the conflicting/broken package has been removed from the tree? Or is there some grace period where they should be kept around?

Reply:
`pkgcheck` warns about outdated blockers when they refer to a package/version that was removed two or more years ago. That seems like a fine guideline to me.